sw/device/lib/testing/pinmux_testutils.c

```diff
@@ -62,6 +62,18 @@ void pinmux_testutils_init(dif_pinmux_t *pinmux) {
   // Configure UART1 TX output to connect to MIO pad IOB5
   CHECK_DIF_OK(dif_pinmux_output_select(pinmux, kTopEarlgreyPinmuxMioOutIob5,
                                         kTopEarlgreyPinmuxOutselUart1Tx));
+
+  // TODO: We have a pull up resistor on the USB_P line, and we need extra drive
+  // strength in dvsim
```

Comment:
I think this is fine for simulation purposes.

We only have [4 driving strength levels](https://www.hdlworks.com/hdl_corner/verilog_ref/items/Strengths.htm#:~:text=Verilog%20has%204%20driving%20strengths,strength%20of%20the%20strongest%20driver.) available in SV to model pad driving strengths and pullups, and we intentionally did not use supply strength, which leaves us with 3 levels.

So I think it is ok to remove the TODO keyword here and explain the use of weak, pull and strong in the simulation model for clarity.